lib/scrub/cip/connection_manager.ex

```diff
@@ -127,16 +141,16 @@ defmodule Scrub.CIP.ConnectionManager do
     request_path_padded = <<request_path::binary, 0x00::size(request_path_padding)-unit(8)>>
     request_path = <<0x91, byte_size(request_path)::usint, request_path_padded::binary>>
 
-    request_words = (byte_size(request_path) / 2) |> floor
-    read_elements = opts[:read_elements] || 1
+    <<request_path::binary>>
```

Comment:
Subjective.

It doesn't look like a change was required here, but if we are going to change something here we could do just:

```elixir
  <<0x91, byte_size(request_path)::usint, request_path_padded::binary>>
```

Usually I see what is in the PR here when there is semantic/readability value to adding in the variable name but IMO I don't see that here.